Automattic/wp-calypso
# State: Introduce connected application actions

This PR introduces actions and action creators for connected applications:

* `requestConnectedApplications`
* `receiveConnectedApplications`
* `deleteConnectedApplication`
* `deleteConnectedApplicationSuccess`

Tests are included.

For context of how these actions and action creators would be used, you can see the full connected applications reduxification PR: #24175.

To test:
* Checkout this branch.
* Verify all tests pass:

```
npm run test-client client/state/connected-applications/test/actions.js
```
